ref/general/os-hardening.adoc

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: Operating system security hardening
+:seo-description:
+:page-layout: general-reference
+:page-type: general
+= Operating system security hardening
+
+Operating system (OS) intrusions occur when unauthorized users can log in to the OS and gain access to sensitive resources.
+Before you run a container in production, harden the container image to remove any attack vectors that might allow exploitation of your server's configuration or install.
+
+== Open Liberty Docker images
+A container that runs in production should have only the tools and packages that you need to run your application. You can find Open Liberty Docker images and information about them in link:https://hub.docker.com/_/open-liberty[Docker Hub].
+
+=== Use an existing Open Liberty Docker image
+The simplest way to securely run Open Liberty is by using existing Open Liberty Docker images, which are already secured and have OS permissions set up.
+If you want to extend an Open Liberty image for use in production, opt for the leanest possible configuration by adding only tools and packages that are essential to your application.
+Using a minimal image reduces the number of OS vulnerabilities that you bundle with your application.
+
+=== Build and secure an Open Liberty Docker image
+If you want to build your own Open Liberty Docker image, you need to secure and harden the image before you use it in production.
+
+// Confirm the following things about your Open Liberty image before it runs in production:
+
+// * ONE
+
+== Stay current with Open Liberty
+Keep your version of Open Liberty up to date.
+Open Liberty's zero migration policy means that you can move to a later version without changing anything in your server configuration.
+Staying on the latest version also ensures that you get the latest link:/docs/ref/general/#security-vulnerabilities.html[security fixes] because updates are first released in the current version.
+After security fixes are released for the current version of Open Liberty, they are rolled back to older releases.
+
+== Validate installation
+You can validate your Open Liberty server installation by running the `/<wlp_install_dir/bin/productInfo validate` command.
+This command uses an MD5 checksum to check your installation for errors or potential vulnerabilities.
+The output message that you receive indicates whether Open Liberty was validated successfully.
+
+== UNIX file permissions
+A critical aspect of OS security is ensuring that users have the correct permissions.
```

Comment:
Change,
"A critical aspect of OS security is ensuring that users have the correct permissions.
This means that users must fall into the appropriate UNIX permissions bit."
to,
"A critical aspect of OS security is ensuring that users have the correct permissions,
which means that users must fall into the appropriate UNIX permissions bit."